qemu/tests/ovs_enslave_port.py

```diff
@@ -0,0 +1,65 @@
+from avocado.utils import process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Enslave a port which is already a member of ovs into another
+
+    1) Check current ovs bridge and list the ports in it
+    2) Create a new ovs bridge
+    3) Enslave one port which is a member of current ovs bridge into the new one
+    4) Delete the new ovs bridge
+
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_ovs_ports(name):
```

Comment:
We have too many these similar functions, I would recommend to have a base module for it, including all this basic functions there, and all ovs related tests might import the module.